setup.py

```diff
@@ -25,10 +27,14 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(os.path.abspath(__file__)), fname)).read()
 
+def get_version():
+    with open("convert2rhel/__init__.py") as f:
+        version = re.findall(r'^__version__ = "([^"]+)"\r?$', f.readline(), re.M)[0]
```

Comment:
This can be `re.search`

Reply:
yeap, no preferences in either solution. But I am lazy:)

Reply:
actually search returns the whole matching string, but I need a capturing group, so findall is better